src/server/rdb_save.cc

```diff
@@ -1222,19 +1263,19 @@ RdbSaver::RdbSaver(::io::Sink* sink, SaveMode save_mode, bool align_writes) {
     case SaveMode::SINGLE_SHARD:
     case SaveMode::SINGLE_SHARD_WITH_SUMMARY:
       producer_count = 1;
-      if (compression_mode == 3) {
-        compression_mode_ = CompressionMode::MULTY_ENTRY_LZ4;
-      } else if (compression_mode == 2) {
-        compression_mode_ = CompressionMode::MULTY_ENTRY_ZSTD;
-      } else if (compression_mode == 1) {
+      if (compression_mode == CompressionMode::MULTI_ENTRY_LZ4) {
```

Comment:
I think you can just write compression_mode_ = compression_mode instead of lines 1266-1273

Reply:
wow I'm such a robot, nice :)

Reply:
I am trying to think if you are AI bot